Provider/src/FirebirdSql.EntityFrameworkCore.Firebird/Utilities/FbSettings.cs

```diff
@@ -0,0 +1,81 @@
+﻿/*
+ *    The contents of this file are subject to the Initial
+ *    Developer's Public License Version 1.0 (the "License");
+ *    you may not use this file except in compliance with the
+ *    License. You may obtain a copy of the License at
+ *    https://github.com/FirebirdSQL/NETProvider/blob/master/license.txt.
+ *
+ *    Software distributed under the License is distributed on
+ *    an "AS IS" basis, WITHOUT WARRANTY OF ANY KIND, either
+ *    express or implied. See the License for the specific
+ *    language governing rights and limitations under the License.
+ *
+ *    All Rights Reserved.
+ */
+
+//$Authors = Jiri Cincura (jiri@cincura.net), Rafael Almeida (ralms@ralms.net)
+
+using System;
+using System.Collections.Concurrent;
+using System.Data;
+using System.Data.Common;
+using FirebirdSql.Data.FirebirdClient;
+
+namespace FirebirdSql.EntityFrameworkCore.Firebird.Utilities
+{
+
+	public class FbSettings
+	{
+
+		public Version ServerVersion;
+		public bool IsSupportIdentityIncrement => ServerVersion.Major >= 3;
+		public int ObjectLengthName => ServerVersion.Major == 3  ? 31 : 64;
```

Comment:
On FB3 it's 63 max. And it's configurable. So you really don't know until you try...